query.go

```diff
@@ -944,13 +959,25 @@ func (q *MultiPhraseQuery) Field() string {
 	return q.field
 }
 
+// Slop returns the acceptable distance between terms
+func (q *MultiPhraseQuery) Slop() int {
+	return q.slop
+}
+
+// SetSlop updates the sloppyness of the query
+// the phrase terms can be as "dist" terms away from each other
```

Comment:
I think we should be careful describing the sloppyness parameter, I think in some cases it can be harder to explain that just "distance", particular when terms are interchanged.  I don't know what to suggest, and this is fine as a starting point I think.

Reply:
Yeah, I wasn't sure how precise i should be here, so i went with vague. 😅